README.md

```diff
@@ -60,22 +60,39 @@ A third-party plugin is required to enable syntax coloring. The Che4z basic stac
 	
 ## Copybook Support
 
-The LSP for COBOL extension can retrieve copybooks used in your projects from the mainframe and download them locally. You can open copybooks in your IDE and make use of the copybook support features of the extension.
+The COBOL Language Support extension can retrieve copybooks used in your projects from the mainframe and download them locally. You can also store copybooks locally in folders in your workspace. Copybook support features of the extension are enabled whenever a copybook called in source code is successfully located in a specified mainframe data set or local folder.
 
-#### Retrieving Copybooks
+Ensure that copybook names are specified in your source code in upper case, for example `COPY BOOK1` and not `COPY book1`.
+
+### Retrieving Copybooks from the Mainframe
 
 To set up automatic copybook retrieval from the mainframe, **follow these steps:**
 
 1. Ensure that you have a [Zowe CLI z/OSMF profile](https://docs.zowe.org/stable/user-guide/cli-configuringcli.html) configured, with credentials defined.
 2. Open the **Extensions** tab, click the cog icon next to **COBOL Language Support** and select **Extension Settings** to open the COBOL Language Support extension settings. 
-3. Under **Paths**, list the names of any number of partitioned data sets on the mainframe to search for copybooks. The data sets are searched in the order they are listed, so if two data sets contain a copybook with the same member name, the one from the data set higher on the list is downloaded.
+3. Under **Paths: Dsn**, list the names of any number of partitioned data sets on the mainframe to search for copybooks. The data sets are searched in the order they are listed, so if two data sets contain a copybook with the same member name, the one from the data set higher on the list is downloaded.
 4. Under **Profile**, enter the name of your Zowe CLI z/OSMF profile.
 5. Open a program or project.  
-   All copybooks used in the program or project which are not stored locally are downloaded from the mainframe data sets that you specified in step 3. Copybooks are stored locally in a **.copybooks** directory within the workspace, which is created automatically.
+   All copybooks used in the program or project which are not stored locally are downloaded from the mainframe data sets that you specified in step 3. Copybooks are stored locally in a **.copybooks** directory within the workspace, which is created automatically.  
```

Comment:
do we want to be more specific here? It gets saved under:
.c4z/.copybooks/<profile>/<DSN>/

Reply:
has this been changed? the workspace I created in 0.11.1 has the mainframe copybooks saved in /.copybooks which isn't a subfolder of /.c4z.

Reply:
you need the "next" version...sent on slack

Reply:
Lukas, this is what I meant when I asked "has this changed". Because if it has, there are still going to be workspaces out there created on 0.11.1 and before with the .copybooks folder in a different location.
So no, I don't think we want to be more specific.

Reply:
I've expanded on this in the "retrieving copybooks" section.